times.py

```diff
@@ -18,9 +22,12 @@ def compute_overlap_time(range1, range2):
             low = max(start1, start2)
             high = min(end1, end2)
             overlap_time.append((low, high))
```

Comment:
⚠️ bug here 🐛 ⚠️ !
Should we add an overlap to the `overlap_time` list if `low > high`?